### DISCUSSION TITLE
SSL Manager version 4.4.1 has breaking change.

### DISCUSSION THREAD

**Discussion by mcswaip1** [nextgenhealthcare/connect#6028] [Q&A]
Hi All!

I was working to use the openapi to communicate with Mirth.
The code was calling "/extensions/ssl/all" passing in the correct format and worked great in 4.4.0. However, apparently there was a breaking change in 4.4.1 that changed the accepted format of the message.

Should I create a Issue in the connect repo or is there another more appropriate repo that I can post the issue?
Thank you!

**Comment by pacmano1** [comment]
if you’re calling the mirth rest API, and it looks like the response changed, and the release notes do not indicate that the response changed, that is likely worth posting an issue in the issue section with complete details.   But I would certainly post it here first and make sure the second set of eyes looks at it before you post it as an issue

**Reply by mcswaip1** [reply]
Good idea. I'm not even sure where to get the release notes for the commercial extensions...

**Reply by pacmano1** [reply]
Can you post an example of the problem?

**Reply by pacmano1** [reply]
Not sure on release notes either, I looked on  client portal and I see the manual, but not release notes.   I was under the impression that this extension had not changed in years.
